### PR TITLE
Added 5 soil SLDs

### PR DESCRIPTION
**Pull request by Kate-Lyndegaard**

### CODE REVIEW

**Review by JohannaOtt — COMMENTED** [review]
I can create the configs with the new layers which is awesome! Unfortunaltey, they are not displayed on the test page of my local service publisher 
grafik
But that can also have various other reasons. I can test on the test system tomorrow if you didn't do so anyway already.

**Comment by Kate-Lyndegaard** [comment]
> 
> 
> I can create the configs with the new layers which is awesome! Unfortunaltey, they are not displayed on the test page of my local service publisher
> grafik
> But that can also have various other reasons. I can test on the test system tomorrow if you didn't do so anyway already.

I am testing it now on test:-)

**Comment by JohannaOtt** [comment]
Is it expected that there are 4 SoilDerivedObject layers? https://test.haleconnect.de/features/#/map?request=https:%2F%2Ftest.haleconnect.de%2Fows%2Fservices%2Forg.8.4f6b8bb6-cf91-4154-b678-8b3a24f3ab12_wms
grafik

For both of my test datasets, I cannot see the geometries in the Map view: https://test.haleconnect.de/#/dataset/org/8/043af7c9-de33-401c-a833-2c4e32f3b2ca/viewServices and https://test.haleconnect.de/#/dataset/org/8/4f6b8bb6-cf91-4154-b678-8b3a24f3ab12/overview

**Comment by JohannaOtt** [comment]
@Kate-Lyndegaard Did you close this PR on purpose? So new further review/testing required?

**Comment by JohannaOtt** [comment]
@Kate-Lyndegaard This PR was closed without being merged. Was that on purpose?

File: config.style [hunk 1]
```diff
@@ -1107,6 +1107,86 @@ Layer{
 	SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/tn-ro/4.0/:RoadNode
 	Styles: TN_RO_RoadNode
 }
+Layer{
```
Comment (JohannaOtt) [inline config.style:1110]: This layer seems to be a mix from ObservedSoilProfile and SoilPlot - @Kate-Lyndegaard is that expected?

**Comment by Kate-Lyndegaard** [comment]
@JohannaOtt These were created for a Brandenburg deployment. They should not be merged with hc.

File: config.style (hunk 1 above)
Comment (Kate-Lyndegaard) [reply config.style:1110]: @JohannaOtt Please check the specification:
grafik

**Pull request closed without merging**